functions.php

```diff
@@ -0,0 +1,22 @@
+<?php 
+
+//  * Takes the data from the database, separates the array into individual players stats and then echos
+//  * out the information while checking that the array is populated 
+//  *
+//  * @param array $players
+//  * @return echos out the details from the player
+//  */
+function displayPlayers(array $players){
```

Comment:
You can return type hint this function as string

Reply:
good idea, ive added this